docs/swarm-api.md

```diff
@@ -19,11 +19,8 @@ Remote API.
 
 ## Missing endpoints
 
-Some endpoints have not yet been implemented and will return a 404 error.
-
-```
-POST "/images/create" : "docker import" flow not implement
-```
+Now Docker API that Swarm supports is version 1.22 or lower. In API version 1.22,
```

Comment:
Please edit for clarity:

As of Swarm version 1.2.1, Swarm supports the API version 1.22 or lower. There are no missing endpoints for Swarm in API version 1.22.